setup.py

```diff
@@ -15,5 +15,6 @@
     install_requires=[
                       'GitPython>=2.1.3',
                       'PyYAML>=3.12',
+                      'click>=6.7',
```

Comment:
Not sure why this is still showing an update. It's fixed in master already. Maybe GH bug?

https://github.com/iamseth/saltstack-vulcan/blob/master/setup.py